### DISCUSSION TITLE
Kiai glow is missing on slider bodies in legacy skins

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#15331] [Q&A]
While the hitcircle flashing during kiai time in legacy skins has been added a while back, lazer lacks flashing the sliderbody and the hit lighting flashing (i.e hit lighting appears around the hitcircle, even before its hit) during kiai time, like stable does.

It's not really a big deal, as they dont do much other than look pretty, but with the right skin, they look really pretty, so It'd be a nice to have feature :)

**Comment by peppy** [comment]
That sounds weird, as it does look to be implemented. Have you tried with the osu!classic skin?

**Reply by 2minwia** [reply]
(I'm noticing I completely missed the "reply" button here, so I'm just copy-pasting my "suggested answers" (deleted now) as a reply here.)

It's way harder to notice that It's not there with the osu!classic skin, as it uses smaller hitlighting and white sliderbodies, mostly masking the flashes, but the kiai flashes are still missing in lazer. (Other than, again, the hitcircle flashing.)

I made a comparison video to show this with the osu!classic skin and with a skin where It's more obvious: https://streamable.com/3okbtz

**Reply by bdach** [reply]
Looking at the source it does appear that there's not really any chance that slider bodies have kiai flashes applied to them... If I'm looking correctly it will indeed only show on the circles as-is.
